development/docs.rst

```diff
@@ -61,83 +55,79 @@ This is rendered in the documentation as follows:
     2
 
 
-While specifying code examples under the **.. doctest::** directive,
-either all statements must be specified by the **>>>** prompt or without it. For
-every prompt, any potential corresponding output must be specified immediately
-after it. This directive is ideally used when there are a number of examples that
-need to be checked in quick succession.
+While specifying code examples under the ``.. doctest::`` directive, either all statements must be specified by the ``>>>`` prompt or without it.
+For every prompt, any potential corresponding output must be specified immediately after it.
+This directive is ideally used when there are a number of examples that need to be checked in quick succession.
+
+A different way to specify code examples (and test them) is using the associated ``.. testcode::`` directive which is effectively a code block: ::
 
-A different way to specify code examples (and test them) is using the associated
-**.. testcode::** directive which is effectively a code block:
-::
     .. testcode::
 
         a = 2
         print(a)
 
-followed by  it's results. The result can be specified with the
-**.. testoutput::** block:
-::
+followed by its results.
+The result can be specified with the ``.. testoutput::`` block: ::
+
     .. testoutput::
 
         2
 
-The advantage of the **testcode** directive is that it is a lot simpler to
+The advantage of the ``testcode`` directive is that it is a lot simpler to
 specify and amenable to copying the code to clipboard. Usually, tests are
 more easily specified with this directive as the input and output are
 specified in different blocks. The rendering is neater too.
 
 .. note::
-    The **doctest** and **testcode** directives should not be assumed to
+    The ``doctest`` and ``testcode`` directives should not be assumed to
     have the same namespace.
 
+**Output:**
+
 .. testcode::
 
     a = 2
     print(a)
 
-**Output:**
```

Comment:
Why did the "Output" heading move to above the code? Perhaps the heading name should change? Or perhaps there should be two headings?

Reply:
I read this "output" as meaning "render" - these two blocks are the rendered output of the example code that's ~a paragraph above.  Not _certain_ that's correct, but it looks right to me in the rendered documentation.

Reply:
![Uploading Screenshot 2021-04-10 at 23.29.21.png…]()

Reply:
Looks good! Let's keep it this way.